pkg/bot/slack_cloud.go

```diff
@@ -211,6 +212,12 @@ func (b *CloudSlack) start(ctx context.Context) error {
 	b.setFailureReason("")
 	go b.startMessageProcessor(ctx, messageWorkers, messages)
 
+	if err := b.reporter.ReportBotEnabled(b.IntegrationName(), b.commGroupMetadata.Index); err != nil {
```

Comment:
As you know, retries can occur quite often. Would it be a problem to report multiple bot-enabled events? Should we use `sync.Once` here? This way, it will trigger only during each Botkube restart, which happens a bit less often.

Reply:
No, that's not a problem as we already report such events multiple times for the instances that restart. So in the telemetry dashboard we already can't rely on number of such events per instance - instead, we should count such distinct events for platforms.

Anyway, if I'll be already modifying the PR, I can add sync.Once, that's not a problem 👍 it'll reduce the emitted events by tiny bit, so it's still a small improvement.